src/debug/debug-stack-trace-iterator.cc

```diff
@@ -219,6 +219,12 @@ void DebugStackTraceIterator::UpdateInlineFrameIndexAndResumableFnOnStack() {
   iterator_.frame()->Summarize(&frames);
   inlined_frame_index_ = static_cast<int>(frames.size());
 
+  if (recordreplay::IsRecordingOrReplaying() && !frames.size()) {
+    // Replay workaround: Summarize sometimes won't work for an unknown reason.
+    // https://linear.app/replay/issue/RUN-1920
+    return;
```

Comment:
Can we emit a warning when taking this path?  Then it will be more obvious if we have unexpected missing stacks in a pause or something.